src/app/app.js

```diff
@@ -24,8 +23,6 @@ import CONFIG from './config';
 import routes from './routes';
 import '../sass/app.scss';
 
-window.moment = timezone;
```

Comment:
i am not sure if this is equivalent. But importing moment-timezone will cause to bundle it with app. It seems from docu, it works with plain without reassigning moment https://github.com/moment/moment-timezone